src/schema/ACLs/index.js

```diff
@@ -1,14 +1,13 @@
-const AWS_ACCESSKEY_REGEX = "AWS4-HMAC-SHA256 Credential=";
-const OAUTH_ACCESSKEY_REGEX = "Bearer\s";
-const matchAwsToken = str => str.match(/(`${AWS_ACCESSKEY_REGEX}`)/g);
-const matchOAuthToken = str => str.match(/(`${OAUTH_ACCESSKEY_REGEX}`)/g);
+const AWS_ACCESSKEY_REGEX = /(AWS4\-HMAC\-SHA256 Credential\=)/g;
+const OAUTH_ACCESSKEY_REGEX = /(Bearer\s)/g;
+const matchAwsToken = str => str.match(AWS_ACCESSKEY_REGEX);
+const matchOAuthToken = str => str.match(OAUTH_ACCESSKEY_REGEX);
 
 //calls the check the ACLS on the model and return the access permission on method.
 function checkAccess({ req, id, model, method }) {
-  return getAccessToken(req, model)
-    .then(token => {
-      return checkACL(token, id, model, method);
-    });
+  return getAccessToken(req, model).then(token => {
```

Comment:
We discussed changing `getAccessToken` to something like `getContextDetails` so that it can taken in the `req` object and return back the real info for the methods invoked later. Passing `req` all the way down is not recommended.

Reply:
Based on this, the `checkAcl` method signature could be 

    checkACL({token, orgId}, id, model, method)

Reply:
first param should be always access-token for checkACL of loopback. passing the required req params will be the last argument as it matches the loopback function signature. Passing entire req allow us to access other required data like userid.

Reply:
Think of request processing as a pipeline. In case of a typical express handler, every handler is working on the same pipeline, so it needs access to raw request. However, in graphql, the resolver code doesn't need request all the way down. The purpose of getContextDetails is to extract anything of interest from request and make it available for the APIs that are invoked after it.

Reply:
What if the user has no orgId, he is accessing authenticated APIs only which doesn't require orgId. 
Here the function "getAccessToken" is to resolve access-token it 2 different formats. adding orgId resolution which is not even required value it seems invalid. 
we can pass only orgId in an object instead of whole req for checkACL function.